Sources/Datadog/Core/Persistence/Files/File.swift

```diff
@@ -46,60 +46,60 @@ internal struct File: WritableFile, ReadableFile {
     func append(data: Data) throws {
         let fileHandle = try FileHandle(forWritingTo: url)
 
+        // NOTE: RUMM-669
+        // https://github.com/DataDog/dd-sdk-ios/issues/214
+        // https://en.wikipedia.org/wiki/Xcode#11.x_series
+        // compiler version needs to have iOS 13.4+ as base SDK
+        #if compiler(>=5.2)
         if #available(iOS 13.4, *) {
-            /**
-             Even though the `fileHandle.seekToEnd()` should be available since iOS 13.0:
-             ```
-             @available(OSX 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
-             public func seekToEnd() throws -> UInt64
-             ```
-             it crashes on iOS Simulators prior to iOS 13.4:
-             ```
-             Symbol not found: _$sSo12NSFileHandleC10FoundationE9seekToEnds6UInt64VyKF
-             ```
-            */
```

Comment:
Why removing this comment 🤔? It's a valid explanation of the unobvious setup - we use `#available(iOS 13.4, *)` for the API which is marked as `@available(iOS 13.0, *)`. Next time we come here and we may remove it by mistake.

Reply:
it doesn't compile with `#available(iOS 13)` ❌ 
maybe they fix the available versions were wrong in the previous iOS SDK and now fixed

<img width="1011" alt="Screenshot 2020-08-14 at 10 07 08" src="https://user-images.githubusercontent.com/1417500/90228136-00114500-de16-11ea-9651-40c9cfdf41e8.png">
<img width="1009" alt="Screenshot 2020-08-14 at 10 07 26" src="https://user-images.githubusercontent.com/1417500/90228141-01427200-de16-11ea-8dbe-ade366813d88.png">

Reply:
this issue is fixed in iOS 14/Xcode 12, but Xcode 11.6 still has the issue: leaving the comment here

Reply:
👌